Lib/logging/config.py

```diff
@@ -350,6 +358,10 @@ def __getitem__(self, key):
         # Can't replace a tuple entry.
         return self.convert_with_key(key, value, replace=False)
 
+    def __iter__(self):
```

Comment:
Is it needed? Are any tests failed without it?

Reply:
No tests fail without it, but the request is for additional functionality (which would need its own tests, of course).

Reply:
I suppose that tests for iterating ConvertingList and ConvertingTuple would expose that there is no need to add `__iter__()`.

Tests are necessary.